lib/iris/fileformats/_pyke_rules/fc_rules_cf.krb

```diff
@@ -913,7 +913,7 @@ fc_extras
         # Set the cube global attributes. 
         for attr_name, attr_value in six.iteritems(cf_var.cf_group.global_attributes):
             try:
-                if isinstance(attr_value, unicode):
+                if isinstance(attr_value, six.text_type):
```

Comment:
This code path is for converting unicode to str to avoid print-outs using the `u` prefix syntax where possible. That problem doesn't apply in Python 3 so I'd be tempted to use `if six.PY2 and isinstance(attr_value, unicode):` to make that clear.